src/components/SecondaryPanes/SecondaryPanes.css

```diff
@@ -5,6 +5,17 @@
   white-space: nowrap;
 }
 
+/*
+  We apply overflow to the container with the commandbar.
+  This allows the commandbar to remain fixed when scrolling
+  until the content completely ends. Not just the height of
+  the wrapper.
+  Ref: https://github.com/devtools-html/debugger.html/issues/3426
+*/
+.secondary-panes--sticky-commandbar {
```

Comment:
lol - this could have easily been `. secondary-panes.sticky-commandbar`, but you wanted to go all BEM on us :)

I'm okay with this, but just being my morning snarky self :)